Bucketeer2Tests/EvaluationDaoTests.swift

```diff
@@ -0,0 +1,131 @@
+import XCTest
+@testable import Bucketeer2
+
+final class EvaluationDaoTests: XCTestCase {
+    let url = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent("evaluation_test.db")
+    var path: String { url.path }
+
+    let mocks: [Evaluation] = [
+        Evaluation(
+            id: "evaluation1",
+            feature_id: "feature1",
+            feature_version: 1,
+            user_id: "user1",
+            variation_id: "variation1",
+            variation: .init(
+                id: "variation1",
+                value: "value1",
+                name: "name1",
+                description: "description1"
+            ),
+            reason: .init(
+                type: .rule,
+                rule_id: "rule1"
+            ),
+            variation_value: "variation_value1"
+        ),
+        Evaluation(
+            id: "evaluation2",
+            feature_id: "feature2",
+            feature_version: 2,
+            user_id: "user1",
+            variation_id: "variation2",
+            variation: .init(
+                id: "variation2",
+                value: "value2",
+                name: "name2",
+                description: "description2"
+            ),
+            reason: .init(
+                type: .rule,
+                rule_id: "rule2"
+            ),
+            variation_value: "variation_value2"
+        )
+    ]
+
+
+    override func setUp() {
+        super.setUp()
+
+        let db = try! SQLite(path: path, logger: nil)
+
+        let evaluationTable = SQLite.Table(entity: EvaluationEntity())
+        let evaluationSql = evaluationTable.sqlToCreate()
+        try! db.exec(query: evaluationSql)
+
+        let dao = EvaluationDaoImpl(db: db)
+        try! dao.put(userId: "user1", evaluations: mocks)
```

Comment:
Reading the test cases is easier and safe if you don't add data in the setup step.
Doing this doesn't cover cases if the interactor adds a single item or multiple items.

Would you mind covering other cases like in the android tests, please?
The same for EventDaoImplTest.

### Evaluation
https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/evaluation/db/EvaluationDaoImplTest.kt

### Event
https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/event/db/EventDaoImplTest.kt

Reply:
Covered other cases https://github.com/ca-dp/bucketeer-ios-sdk/pull/83/commits/919ab813b4c163e17ae2b678af4d30128f75bd90 with simple mock data https://github.com/ca-dp/bucketeer-ios-sdk/pull/83/commits/c42c76f80da9e40fb60ef3f7dffa5707a009da60 🙆‍♀️